### ISSUE TITLE
How to use deeptables for multilabel problems?

### ISSUE THREAD

**Issue by lmassaron** [DataCanvasIO/DeepTables#14]
I would like to use `DeepTable` for a multilabel problem. I can fix the last layer using a custom layer with as many neurons with sigmoid activation as the number of the labels, but as for as the input, a matrix of y is not accepted.

**Comment by jackguagua** [comment]
Multilabel task are not currently supported.

**Comment by TheEdoardo93** [comment]
Any update on this topic?

**Comment by oaksharks** [comment]
@TheEdoardo93 Now we can support multilabel classification task, here is an example:
```python
from sklearn.model_selection import train_test_split
import numpy as np
import pandas as pd
from deeptables.models import deeptable
from deeptables.datasets import dsutils

x1 = np.random.randint(0, 10, size=(100), dtype='int')
x2 = np.random.normal(0.0, 1.0, size=(100))
x3 = np.random.normal(0.0, 1.0, size=(100))

y1 = np.random.randint(0, 2, size=(100), dtype='int')
y2 = np.random.randint(0, 2, size=(100), dtype='int')

df = pd.DataFrame({'x1': x1, 'x2': x2, 'x3': x3})
df_y = pd.DataFrame({'y1': y1, 'y2': y2})

conf = deeptable.ModelConfig(metrics=['AUC'], nets=['dnn_nets'], apply_gbm_features=False, task='multilabel')
dt = deeptable.DeepTable(config=conf)
X_train, X_test, y_train, y_test = train_test_split(df, df_y, test_size=0.2, random_state=42)
model, history = dt.fit(X_train, y_train.values, epochs=1)
```
You can refer to PR  https://github.com/DataCanvasIO/DeepTables/pull/32 for more.
